test/cypress/integration/simplified-account/service-settings/card-payments/index.cy.js

```diff
@@ -97,23 +74,61 @@ describe('Card payments page', () => {
           cy.get('.govuk-summary-list__row').eq(0).within(() => {
             cy.get('.govuk-summary-list__key').should('contain', 'Collect billing address')
             cy.get('.govuk-summary-list__value').should('contain', 'Off')
+            cy.get('.govuk-summary-list__actions').should('contain', 'Change')
           })
           cy.get('.govuk-summary-list__row').eq(1).within(() => {
             cy.get('.govuk-summary-list__key').should('contain', 'Default billing address country')
-            cy.get('.govuk-summary-list__value').should('contain', 'IE')
+            cy.get('.govuk-summary-list__value').should('contain', 'None')
+            cy.get('.govuk-summary-list__actions').should('contain', 'Change')
           })
         })
         cy.get('.govuk-summary-list').eq(1).within(() => {
           cy.get('.govuk-summary-list__row').eq(0).within(() => {
             cy.get('.govuk-summary-list__key').should('contain', 'Apple Pay')
             cy.get('.govuk-summary-list__value').should('contain', 'Off')
+            cy.get('.govuk-summary-list__actions').should('contain', 'Change')
           })
           cy.get('.govuk-summary-list__row').eq(1).within(() => {
             cy.get('.govuk-summary-list__key').should('contain', 'Google Pay')
             cy.get('.govuk-summary-list__value').should('contain', 'Off')
+            cy.get('.govuk-summary-list__actions').should('contain', 'Change')
           })
         })
       })
+
+      it('should handle view only users', () => {
```

Comment:
The nesting of this should be changed, since it's nested under a `describe('for an admin user')` - it should be moved into a `describe('for a non-admin user')`, and should probably have a few basic tests that the heading and title are correct, as well as testing the content

Reply:
I've rearrange the tests here.